explorer/lib/explorer/periodically.ex

```diff
@@ -44,6 +45,9 @@ defmodule Explorer.Periodically do
     rescue
       error -> IO.puts("An error occurred during batch processing:\n#{inspect(error)}")
     end
+    PubSub.broadcast(Explorer.PubSub, "update_home", %{})
+    PubSub.broadcast(Explorer.PubSub, "update_batch", %{})
+    PubSub.broadcast(Explorer.PubSub, "update_batches", %{})
```

Comment:
I don't think we need different topics, as we can publish a single event notifying the same thing: There has been a data update. Subscribers can choose to listen this topic or not.